tools/doc/html.js

```diff
@@ -165,10 +175,32 @@ function parseLists(input) {
 }
 
 
+// Syscalls which appear in the docs, but which only exist in BSD / OSX
+var BSD_ONLY_SYSCALLS = ['lchmod'];
```

Comment:
Suggestion: Now that we have `Set`s in core, we may be able to speed up this lookup.